src/components/feedback/feedbackModal.tsx

```diff
@@ -188,7 +188,10 @@ export function FeedbackModal({open, onClose, onSubmit}: FeedbackModalProps) {
     });
   };
 
-  const user = window.Sentry?.getCurrentHub().getScope()?.getUser();
+  const user =
+    typeof window.Sentry?.getCurrentHub === 'function'
+      ? window.Sentry?.getCurrentHub().getScope()?.getUser()
+      : null;
```

Comment:
so many `?` in there already... should throw in more instead?

const user = window.Sentry?.getCurrentHub()?.getScope()?.getUser() ?? null;